modules/inline_message/site.js

```diff
@@ -123,8 +128,11 @@ var capture_subject_click = function() {
         var uid = msg_details[0];
         var list_path = msg_details[1];
         var inline_msg_loaded_callback = function() {
-            $('.header_subject th').append('<i class="bi bi-x-lg close_inline_msg"></i>');
-            $('.close_inline_msg').on("click", function() { msg_inline_close(); });
+            if ($('.header_subject th').find('i.bi.bi-x-lg.close_inline_msg').length === 0) {
```

Comment:
This is very specific `find('i.bi.bi-x-lg.close_inline_msg')` do we have many  tag i with class close_inline_msg ?

Reply:
Fixed!